tests/cpp/test_gpu3.cpp

```diff
@@ -9272,6 +9272,83 @@ TEST_F(NVFuserTest, AllIdsMultipleDependencies) {
   }
 }
 
+// Repeating a broadcast ID. RepeatOp should be used.
+TEST_F(NVFuserTest, Repeat1) {
```

Comment:
```suggestion
TEST_F(NVFuserTest, Repeat_Broadcast) {
```
makes the name more meaningful